unit-tests/py/rspy/repo.py

```diff
@@ -0,0 +1,33 @@
+# License: Apache 2.0. See LICENSE file in root directory.
+# Copyright(c) 2021 Intel Corporation. All Rights Reserved.
+
+import os
+
+# this script is located in librealsense/unit-tests/py/rspy, so main repository is:
+source = os.path.dirname( os.path.dirname( os.path.dirname( os.path.dirname( os.path.abspath( __file__ )))))
```

Comment:
I think you meant to call this `librealsense`? See line 15...

Reply:
I personally prefer `root`

Reply:
Do you think `repo` is better or `librealsense` for the module name?

Reply:
I think repo is better, and i'll change source to root